src/xcode/ENA/ENA/Source/Coordinator.swift

```diff
@@ -24,6 +24,7 @@ class Coordinator: RequiresAppDependencies {
 	private weak var delegate: CoordinatorDelegate?
 
 	private let rootViewController: UINavigationController
+	private let contactDiaryStore: DiaryStoringProviding
```

Comment:
```suggestion
	private let contactDiaryStore: DiaryStoring
```

Reply:
Would not build, because `typealias DiaryStoringProviding = DiaryStoring & DiaryProviding` 🤷‍♂️